<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ALUWindow</class>
 <widget class="QMainWindow" name="ALUWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALUWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Sumar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Multiplicar">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dividir">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Componente1Real">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Componente 1</string>
    </property>
    <widget class="QTextEdit" name="NumeroReal_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Real</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IEE754</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hexadecimal</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Componente1IEEE">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Componente 1</string>
    </property>
    <widget class="QTextEdit" name="NumeroIEEE_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Componente1Hex">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Componente 1</string>
    </property>
    <widget class="QTextEdit" name="NumeroHex_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Componente1Real_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Componente 2</string>
    </property>
    <widget class="QTextEdit" name="NumeroReal_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Componente1IEEE_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Componente 2</string>
    </property>
    <widget class="QTextEdit" name="NumeroIEEE_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Componente1Hex_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Componente 2</string>
    </property>
    <widget class="QTextEdit" name="NumeroHex_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SolucionReal">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Solución</string>
    </property>
    <widget class="QLabel" name="SolReal">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SolucionIEEE">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Solución</string>
    </property>
    <widget class="QLabel" name="SolIEEE">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SolucionHex">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>400</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Solución</string>
    </property>
    <widget class="QLabel" name="SolHex">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="info">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>520</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Info.</string>
    </property>
   </widget>
   <widget class="QLabel" name="size">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>590</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="mantisa">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="mantisaSize">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>580</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="exponente">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>510</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="parteFracc">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>580</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="signo">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>100</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>913</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
